### PR TITLE
Fdg 5923

### PR DESCRIPTION
**Pull request by fzafrb**
resolves:
https://federal-spending-transparency.atlassian.net/browse/FDG-5923
89% line coverage

### CODE REVIEW

File: src/transform/explainer-pages-config.js [hunk 1]
```diff
@@ -61,9 +61,9 @@ const explainerPagesSource = {
   },
   "government-revenue": {
     slug: "/americas-finance-guide/government-revenue/",
-    breadCrumbLinkName: "Revenue",
+    breadCrumbLinkName: "Government Revenue",
     seoConfig: {
-      pageTitle: "Government Revenue",
+      pageTitle: "Government Revenue | U.S. Treasury Fiscal Data",
```
Comment (jsmend0za-frb) [inline src/transform/explainer-pages-config.js:66]: You don't need the | U.S. Treasury Fiscal Data this is automatically appended on build.

File: static/data/glossary/glossary.csv [hunk 2]
```diff
@@ -5,27 +5,30 @@ id,term,site_page,definition,url_display,url_path
...
+13,Expenditures,Revenue Explainer,"Government expenditure refers to the purchase of goods and services by the government. Such purchases can include public investments and expenses, as well as disbursements and transfer payments consisting of income transfers (i.e. pensions, social benefits) and capital transfer (i.e. construction grants).",,
+14,Federal Debt,Spending Explainer,The federal debt is the total amount of outstanding borrowing by the U.S. federal government accumulated over the nation�s history. ,,
+15,Fiscal Year,"Debt, Revenue & Spending explainer","The fiscal year is an accounting period spanning 12 months. For the federal government, it runs from October 1 to September 30. For example, Fiscal Year 2022 (FY 2022) started October 1, 2021, and ends September 30, 2022. A fiscal year may be broken down into quarters. For the federal government these quarters are: Q1: October � December, Q2: January � March, Q3: April � June, and Q4: July � September.",,
```
Comment (jsmend0za-frb) [inline static/data/glossary/glossary.csv:16]: There are several ? symbols in this file, that I think are a result of copying and pasting from a Word Document.

Comment (fzafrb) [reply static/data/glossary/glossary.csv]: this glossary.csv file comes from here: https://boozallen.sharepoint.com/teams/pmefiscal_service_ocdo/Shared%20Documents/Forms/AllItems.aspx?cid=4ad4f783%2De6ec%2D4e00%2Daa38%2D6d92174c7385&csf=1&e=7zULkN&FolderCTID=0x012000221E4E25D4286048AA3DE6D8C20D4B3A&id=%2Fteams%2Fpmefiscal%5Fservice%5Focdo%2FShared%20Documents%2F02%20Data%20Strategy%2F01%20FDG%2FGlossary&noAuthRedirect=1&viewid=e040182d%2D14e3%2D4e09%2D93a7%2D417f39ad3a55&web=1

Comment (fzafrb) [reply static/data/glossary/glossary.csv]: I'm not sure where those special characters were coming from, but I just removed/replaced them

File: src/layouts/explainer/sections/government-revenue/sources-of-federal-revenue/sources-of-federal-revenue.jsx [hunk 3]
```diff
@@ -1,95 +1,175 @@
...
+        Government revenue also comes from payments to federal agencies like the
+        U.S. Department of the Interior. Have you visited a national park
+        recently? Did you know your national park entry is included in
+        government revenue? Other agencies generate revenue from leases, the
+        sale of natural resources, and various usage and licensing fees.
+      </p>
+      <div className={visWithCallout}>
+        <SourcesOfRevenueCircleChart />
+        <VisualizationCallout color={revenueExplainerPrimary}>
+          <p>
```
Comment (jsmend0za-frb) [inline src/layouts/explainer/sections/government-revenue/sources-of-federal-revenue/sources-of-federal-revenue.jsx:160]: This label is already in the actual chart component

Comment (fzafrb) [reply src/layouts/explainer/sections/government-revenue/sources-of-federal-revenue/sources-of-federal-revenue.jsx:160]: ah, got it, must have been included in a recent merge.

**Comment by jsmend0za-frb** [comment]
Matches MAC up to the Post-MVP items.
All test passes locally
Builds locally

**Pull request merged**